ci/install_libseccomp.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+#
+# Copyright 2021 Sony Group Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -o errexit
+set -o nounset
+
+arch=$(uname -m)
+
+libseccomp_version="2.5.1"
+libseccomp_tarball="libseccomp-${libseccomp_version}.tar.gz"
+libseccomp_url="https://github.com/seccomp/libseccomp/releases/download/v${libseccomp_version}/${libseccomp_tarball}"
+libseccomp_install_dir="$1"
+cppflags="-O2"
+
+gperf_version="3.1"
+gperf_tarball="gperf-${gperf_version}.tar.gz"
+gperf_url="https://ftp.gnu.org/gnu/gperf/${gperf_tarball}"
+gperf_install_dir="$2"
+
+# We need to build the libseccomp library from sources to create a static library for the musl libc.
+# However, ppc64le and s390x have no musl targets in Rust. Hence, we do not use the musl libc.
+if ([ "${arch}" != "ppc64le" ] && [ "${arch}" != "s390x" ]); then
+    # Set FORTIFY_SOURCE=1 because the musl-libc does not have some functions about FORTIFY_SOURCE=2
+    cflags="-U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=1 -O2"
+fi
+
+finish() {
+    rm -rf "${libseccomp_tarball}" "libseccomp-${libseccomp_version}" "${gperf_tarball}" "gperf-${gperf_version}"
+}
+
+trap finish EXIT
+
+build_and_install_gperf() {
+    echo "Build and install gperf version ${gperf_version}"
+    sudo mkdir -p "${gperf_install_dir}"
+    curl -sLO "${gperf_url}"
+    tar -xf "${gperf_tarball}"
+    pushd "gperf-${gperf_version}"
+    ./configure --prefix="${gperf_install_dir}"
+    make
+    sudo make install
+    export PATH=$PATH:"${gperf_install_dir}"/bin
+    popd
+    echo "Gperf installed successfully"
+}
+
+build_and_install_libseccomp() {
+    echo "Build and install libseccomp version ${libseccomp_version}"
+    sudo mkdir -p "${libseccomp_install_dir}"
+    curl -sLO "${libseccomp_url}"
+    tar -xf "${libseccomp_tarball}"
+    pushd "libseccomp-${libseccomp_version}"
+    ./configure --prefix="${libseccomp_install_dir}" CFLAGS="${cflags}" --enable-static
+    make
+    sudo make install
+    popd
+    echo "Libseccomp installed successfully"
+}
+
+main() {
+    # gperf is required for building the libseccomp.
+    build_and_install_gperf
+    build_and_install_libseccomp
+}
+
+main
```

Comment:
Missing `$@` for the two temporary directories passed to the script in `static-checks.yaml`?

Reply:
I think that it is not necessary because the arguments are referenced directly at the following points.

`libseccomp_instal_dir`
https://github.com/kata-containers/kata-containers/pull/1788/files#diff-a503dc9420524f1744ca0bf10860e00742fd1b8e131ba412cc3a255a67e776b6R23

`gperf_install_dir`
https://github.com/kata-containers/kata-containers/pull/1788/files#diff-a503dc9420524f1744ca0bf10860e00742fd1b8e131ba412cc3a255a67e776b6R31